firmware/lm32/opsis_eeprom.c

```diff
@@ -0,0 +1,144 @@
+#include <generated/csr.h>
```

Comment:
Need to figure out a way to share I2C code. This code is repeated like 5 times now. Created https://github.com/timvideos/HDMI2USB-misoc-firmware/issues/170